src/plugins/d3/renderer/hooks/useShapes/styles.scss

```diff
@@ -40,3 +40,9 @@
         pointer-events: none;
     }
 }
+
+.chartkit-d3-waterfall {
+    &__connector {
+        stroke: var(--g-color-line-generic-accent);
```

Comment:
This element is quite difficult to see in a dark theme
![image](https://github.com/gravity-ui/chartkit/assets/26501073/0ab2df8f-0410-4578-9437-2d0c71fdeeee)